daemon/cmd/endpoint.go

```diff
@@ -595,7 +595,7 @@ func putEndpointIDHandler(d *Daemon, params PutEndpointIDParams) (resp middlewar
 		return api.Error(code, err)
 	}
 
-	ep.Logger(daemonSubsys).Info("Successful endpoint creation")
+	ep.Logger(daemonSubsys).Infof("Successful endpoint creation. UUID: %s", r.uuid)
```

Comment:
Please use structure logging i.e. WithField instead.

Also, it seems like you are deriving the duration of endpoint creation and deletion based on this uuid, do we need to do the same for `cni del` ?